config/locales/vi.yml

```diff
@@ -35,3 +35,6 @@ vi:
       link: "Trang thông tin"
   login:
     title: "Đăng nhập"
+  users:
+    title: "Đăng ký"
+    header: "<h1>Đăng ký</h1>"
```

Comment:
Em dùng đển dịch thôi nhé đừng lưu thẻ html

Reply:
Em sửa lại rồi ạ. Nhưng anh cho em hỏi là sao mình không dùng link_to raw để lấy thẻ html ạ?

Reply:
Ví dụ như header em muốn nó hiện **Modeling** _users_ thì phải làm sao ạ

Reply:
Về câu hỏi tại sao không dùng link_to raw thì do raw là một hàm không excape chuỗi vì thế có thể dẫn đến tấn công xss nếu sử dụng không cẩn thận (tham khảo thêm tại: https://viblo.asia/p/xss-attacks-in-rails-rEBRAaKVR8Zj)
Nếu em muốn thể hiện như vậy thì em sẽ tách và dịch từng chữ một rồi ở view gọi đến thôi. Nếu sau này cần đặt class hoặc chỉnh style khác cũng tiện hơn là lưu cả thẻ html em nhé

Reply:
vâng em hiểu rồi ạ

Reply:
Với trường hợp link_to thì em có thể kết hợp thêm dùng do end để có thể thêm nhiều dòng ở bên dưới nữa nhé

Reply:
anh có thể ví dụ cho em đoạn này link to ntn được k ạ
<a href="abc">This is<em>sample app</em>.</a>

Với cả nếu còn các fix nhỏ a cmt e sửa r merge luôn ạ.

Reply:
Nếu còn các fix nhỏ a cmt e sửa r merge luôn ạ.